### PR TITLE
ssh-to-pgp: init at 1.0.0

### PR DESCRIPTION
**Pull request by Mic92**
This is needed for onboarding sops-nix.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Mic92** [comment]
Me and the sops-nix users tested this tool for quite a while on NixOS 20.09 and on macOS. Therefore I would like to backport it after being merged.

**Review by 0x4A6F — APPROVED** [review]

**Comment by 0x4A6F** [comment]
Result of `nixpkgs-review pr 111543` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>ssh-to-pgp</li>
  </ul>
</details>

File: pkgs/tools/security/ssh-to-pgp/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  '';
+
+  doCheck = true;
+
+  meta = with lib; {
+    description = "Convert ssh private keys to PGP";
+    homepage = "https://github.com/Mic92/ssh-to-pgp";
+    license = licenses.mit;
+    maintainers = with maintainers; [ mic92 ];
+    platforms = platforms.unix;
```
Comment (SuperSandro2000) [inline pkgs/tools/security/ssh-to-pgp/default.nix:28]: This is set by buildGoModule to the correct value.

Comment (Mic92) [reply pkgs/tools/security/ssh-to-pgp/default.nix:28]: https://github.com/NixOS/nixpkgs/pull/111555

**Pull request merged**